src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -1205,6 +1205,7 @@ export class SettingTreeRenderers {
 			new Separator(),
 			this._instantiationService.createInstance(CopySettingIdAction),
 			this._instantiationService.createInstance(CopySettingAsJSONAction),
+			new Separator(),
```

Comment:
This shouldn't be added unless the action after it is present. Otherwise there will be a lone separator at the end of the list (at least on windows with the custom context menu)